apps/organisations/forms.py

```diff
@@ -177,7 +162,28 @@ def __init__(self, *args, **kwargs):
         for lang_code in self.languages:
             for name, field_cls, kwargs in self.translated_fields:
                 self.instance.set_current_language(lang_code)
+                print(field_cls)
                 field = field_cls(**kwargs)
+                # The CKEditor5Widget doesn't have a label field, so we need to set it
+                # after creating the object
```

Comment:
@goapunk I had to bring below also `help_text`, and the plugins fields, or code failed. Now there is still an error message that has to do with django forms:
```
self = <OrganisationForm bound=False, valid=False, fields=(title;logo;image;image_copyright;url;twitter_handle;facebook_handl...tion;nl__slogan;nl__information;ky__description;ky__slogan;ky__information;ru__description;ru__slogan;ru__information)>
name = 'en__information'

    def __getitem__(self, name):
        """Return a BoundField with the given name."""
        try:
            field = self.fields[name]
        except KeyError:
            raise KeyError(
                "Key '%s' not found in '%s'. Choices are: %s." % (
                    name,
                    self.__class__.__name__,
                    ', '.join(sorted(self.fields)),
                )
            )
        if name not in self._bound_fields_cache:
>           self._bound_fields_cache[name] = field.get_bound_field(self, name)
E           AttributeError: 'CKEditor5Widget' object has no attribute 'get_bound_field'
```